src/service.rs

```diff
@@ -19,6 +21,16 @@ pub struct HcHttpGatewayService {
 #[derive(Debug, Clone)]
 pub struct AppState {
     pub configuration: Configuration,
+    #[allow(
+        dead_code,
+        reason = "This will be used when we start making zome calls"
+    )]
+    pub admin_call: Arc<dyn AdminCall>,
```

Comment:
Just to demonstrate that an AppState can be built in unit tests without requiring a running Holochain